src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/ObjectiveC/Wrappers.PlatformNotSupported.cs

```diff
@@ -0,0 +1,299 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Runtime.Versioning;
+
+namespace System.Runtime.InteropServices.ObjectiveC
```

Comment:
We should remove doc from the "not supported" impl.

Reply:
I'm worried that if we remove the PNSE impl's docs, the API docs won't automatically be ported.  I don't know which build the docs team is planning on using to harvest docs from.

Reply:
I'll leave this for now until we are sure that we can remove them and still get the docs auto-ported.